pallets/pallet-dip-provider/src/lib.rs

```diff
@@ -31,23 +31,18 @@ pub mod pallet {
 	use frame_support::{pallet_prelude::*, traits::EnsureOrigin};
 	use frame_system::pallet_prelude::*;
 	use parity_scale_codec::FullCodec;
-	use sp_io::MultiRemovalResults;
 	use sp_std::fmt::Debug;
 
-	use crate::traits::{IdentityCommitmentGenerator, IdentityProvider, SubmitterInfo};
+	use crate::traits::{IdentityCommitmentGenerator, IdentityProvider, ProviderHooks, SubmitterInfo};
 
+	pub type IdentityProviderOf<T> = <T as Config>::IdentityProvider;
 	pub type IdentityOf<T> = <<T as Config>::IdentityProvider as IdentityProvider<<T as Config>::Identifier>>::Success;
 	pub type IdentityCommitmentVersion = u16;
 
 	pub const LATEST_COMMITMENT_VERSION: IdentityCommitmentVersion = 0;
+	pub const MAX_COMMITMENTS_PER_IDENTITY: u16 = LATEST_COMMITMENT_VERSION + 1;
```

Comment:
What is the purpose of this constant? It appears that it may not be used.

Reply:
Ops, good catch! I removed it. It was meant to put an upper bound on the number of commitments per user, but since it's a double map with (DID + commitment version), there is already an upper bound given by the number of versions supported.

Reply:
https://github.com/KILTprotocol/kilt-node/pull/574/commits/1841f0fa64e49381d4e18d3124950624a8343d65